nilearn/datasets/atlas.py

```diff
@@ -782,3 +782,63 @@ def fetch_coords_dosenbach_2010(ordered_regions=True):
                   networks=out_csv['network'], description=fdescr)
 
     return Bunch(**params)
+
+
+def fetch_atlas_surf_destrieux(data_dir=None, url=None,
+                               resume=True, verbose=1):
+    """Download and load the Destrieux cortical atlas (dated 2009)
```

Comment:
I would write "Download and load Destrieux et al, 2010 cortical atlas." and after one line empty space

I would write "This atlas returns 74 sulco-gyral structures per hemisphere distributed with Freesurfer in fsaverage5 surface space."

I choose to update with 2010 paper. The annotations are described well in this paper. At some point, users might ask about the reference point from where we have picked those annotations.